modules/drivers/mcp230/mcp230.js

```diff
@@ -32,138 +32,104 @@ const PULLUPS_8  = 0b00000000;
 const INPUTS_16  = 0b1111111111111111;
 const PULLUPS_16 = 0b0000000000000000;
 
-// Default Config (Both)
-const ADDRESS = 0x20;
-const HZ = 0x00;
-const SDA = 0x04;
-const SCL = 0x05;
```

Comment:
Removed per *I2C* section of https://github.com/Moddable-OpenSource/moddable/pull/21#issuecomment-353265714